src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1568,6 +1575,8 @@ static void FillScanInfo(Eventinfo *lf,cJSON *scan_id,cJSON *name,cJSON *descrip
             sprintf(value, "%d", invalid->valueint);
         } else if (invalid->valuedouble >= 0) {
             sprintf(value, "%lf", invalid->valuedouble);
+        } else {
```

Comment:
This is an unexpected condition: invalid value. Please print a debugging log here.